transformer_engine/pytorch/attention.py

```diff
@@ -1729,6 +1729,7 @@ def __init__(
         rotary_percent: float = 1.0,
         seq_len_interpolation_factor: Optional[int] = None,
         pretrained_max_position_embeddings: Optional[int] = None,
+        rotary_base: int = 10000,
```

Comment:
As far as I can tell, the base doesn't need to be integer-valued:
```suggestion
        rotary_base: float = 10000.0,
```

Reply:
I think the "base" itself is assumed to be an integer in RoPE but yeah as an argument, it probably doesn't matter much. Since the arange tensor is created as float, end result is a float tensor anyways

Reply:
From what I can tell, 10000 is just a random number that was used in ["Attention Is All You Need"](https://arxiv.org/pdf/1706.03762) and it just stuck. 10000.1 would have worked as well.